### PR TITLE
feat(cli): List available parsers and serializers

### LINKED ISSUE #15425: telegraf plugins parsers does not list available parsers
### Use Case

I want to check whether my custom build includes the parsers I need

### Expected behavior

```shell
telegraf plugins parsers
```
Includes `parsers.json` etc.

### Actual behavior

```shell plugins
telegraf plugins parsers
```
Shows all plugins, including inputs, outputs, secretstores, but no parsers

### Additional info

_No response_

### PR DESCRIPTION
**Pull request by LarsStegman**
## Summary

See #15425 

## Checklist

- [x] No AI generated code was used in this PR

## Related issues

resolves #15425

### CODE REVIEW

File: cmd/telegraf/cmd_plugins.go [hunk 1]
```diff
@@ -3,6 +3,8 @@ package main
 
 import (
 	"fmt"
+	"github.com/influxdata/telegraf/plugins/parsers"
+	"github.com/influxdata/telegraf/plugins/serializers"
```
Comment (powersj) [inline cmd/telegraf/cmd_plugins.go:7]: Can you move these down to the other telegraf inputs? We like to keep the stdlib imports on their own.

**Review by powersj — COMMENTED** [review]
One minor change and this is good to go! Thank you very much!

File: cmd/telegraf/cmd_plugins.go [hunk 2]
```diff
@@ -10,8 +10,10 @@ import (
 	"github.com/influxdata/telegraf/plugins/aggregators"
 	"github.com/influxdata/telegraf/plugins/inputs"
 	"github.com/influxdata/telegraf/plugins/outputs"
+        "github.com/influxdata/telegraf/plugins/parsers"
 	"github.com/influxdata/telegraf/plugins/processors"
```
Comment (powersj) [inline cmd/telegraf/cmd_plugins.go:14]:
```suggestion
	"github.com/influxdata/telegraf/plugins/outputs"
    "github.com/influxdata/telegraf/plugins/parsers"
	"github.com/influxdata/telegraf/plugins/processors"
```

Comment (LarsStegman) [reply cmd/telegraf/cmd_plugins.go]: Shouldn't have tried to do it from my phone hehe.

Comment (powersj) [reply cmd/telegraf/cmd_plugins.go]: Hmm did my proposal not work?

Comment (LarsStegman) [reply cmd/telegraf/cmd_plugins.go]: No, doesn't look like it. Hopefully it works now.

**Review by powersj — APPROVED** [review]
Thanks for jumping on this!

**Review by DStrand1 — APPROVED** [review]

**Review by srebhan — APPROVED** [review]
Thanks @LarsStegman!

**Pull request merged**